src/test/java/uk/gov/hmcts/reform/ccd/document/am/service/impl/DocumentManagementServiceImplTest.java

```diff
@@ -579,6 +542,7 @@ private void verifyCaseDataServiceGetDocMetadata() {
     }
 
     @Test
+    @SuppressWarnings("unchecked")
```

Comment:
can you compare with development and remove these suppressions in this file, for some reason they have come back in the PR's. It happened in mine also